OmniBLE.xcodeproj/project.pbxproj

```diff
@@ -76,7 +76,7 @@
 		84752E9E26ED10A0009FD801 /* CoreBluetooth.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 84752E9D26ED10A0009FD801 /* CoreBluetooth.framework */; };
```

Comment:
The changes to this file at a high level:

- Remove the Embed Frameworks phase, on the OmniBLE target, as I found that caused some issues with adding tests, and was not actually necessary from my investigation. I think we would only need this if we want to add a stand-alone app that runs on a device. In that case I think we would just add a special target for that which embeds the frameworks.